src/pages/[platform]/start/migrate-to-gen2/index.mdx

```diff
@@ -82,28 +140,146 @@ During the migration for the new `migrate` environment, it is recommended to con
 
 Next, prepare your local codebase for migration execution by running the following command:
 
+<Callout info>
+
+**Testers Only**
+
+If you are using a names AWS CLI profile other than `default`, you will need to set the `AWS_PROFILE` environment variable to the name of the profile.
+
+```bash title="Terminal" showLineNumbers={false}
+export AWS_PROFILE=your-profile-name
+```
+
+then run the following command to prepare your local codebase for migration execution:
+
+```bash title="Terminal" showLineNumbers={false}
+npx @aws-amplify/migrate@0.1.0-next-3.0 to-gen-2 prepare
+```
+
+</Callout>
+
 ```bash title="Terminal" showLineNumbers={false}
 npx @aws-amplify/migrate@latest to-gen-2 prepare
 ```
 
 After this command completes successfully your `amplify/` directory is converted to an Amplify Gen 2 backend authored in TypeScript. In the next section we will walk through how to validate the generated code using [Amplify Gen 2's sandbox feature](/[platform]/deploy-and-host/sandbox-environments/setup/).
 
+
 ## Validating the preparation
 
 The preparation command will add npm dependencies relevant to your Gen 2 backend in the root `package.json`. Before you begin with the validation, install dependencies:
 
+
 ```bash title="Terminal" showLineNumbers={false}
 npm install
 ```
 
+<Callout info>
+
+**Testers Only** 
+
+For testing purposes, you will need to additionally install the tagged versions of the `@aws-amplify/backend` and `@aws-amplify/backend-data` packages.
+
+```bash title="Terminal" showLineNumbers={false}
+npm install @aws-amplify/backend@0.0.0-test-20250307211308 @aws-amplify/backend-data@0.0.0-test-20250307211308
+```
+
+</Callout>
+
+<Callout info>
+
+**Testers Only**
+
+**Auth**
+
+We will need to add the following manually to `backend.ts` file to set up the additional user pool client for your `Identity providers`. This is needed since Gen2 only supports 1 user pool client out-of-the box:
```

Comment:
```suggestion
We will need to add the following manually to `backend.ts` file to set up the additional user pool client for your `Identity providers`. This is needed since Gen 2 only supports 1 user pool client out-of-the-box:
```